### PR TITLE
Core 1195 filter pills test

### PR DESCRIPTION
**Pull request by roger-in-kiva**
- new quick filters ui showed under a test

### CODE REVIEW

**Comment by dyersituations** [comment]
Seeing a scrollbar now around the gender pills.

image

File: src/components/LoansByCategory/QuickFilters/LocationSelector.vue [hunk 1]
```diff
@@ -231,7 +230,11 @@ export default {
 		withCategories: {
```
Comment (dyersituations) [inline src/components/LoansByCategory/QuickFilters/LocationSelector.vue:230]: Looks like this prop can be removed now.

File: src/components/LoansByCategory/QuickFilters/QuickFilters.vue [hunk 2]
```diff
@@ -1,10 +1,24 @@
 <template>
 	<div class="tw-flex tw-flex-col tw-mb-2 tw-w-full">
+		<div v-if="!withCategories && !enableFilterPills" class="tw-flex tw-items-center tw-mb-2">
```
Comment (dyersituations) [inline src/components/LoansByCategory/QuickFilters/QuickFilters.vue:3]: Why were these two blocks of new elements needed? I thought the old version of the filters already were shown on LH.

Comment (roger-in-kiva) [reply src/components/LoansByCategory/QuickFilters/QuickFilters.vue:3]: These were removed with ui update, so, if we need to keep design like here https://www.kiva.org/lend-by-category/agriculture for testing, we need these new(old) blocks, right?
Like here https://github.com/kiva/ui/blob/155c55d83dde1f5180c9964ec9130635a4d41fa0/src/components/LoansByCategory/QuickFilters/QuickFilters.vue#L2

Comment (dyersituations) [reply src/components/LoansByCategory/QuickFilters/QuickFilters.vue:3]: If this was how it was before, then makes sense, thanks.

**Review by dyersituations — APPROVED** [review]

**Pull request merged**